src/lib/wsman-xml-serialize.c

```diff
@@ -82,7 +82,7 @@ WsSerializerContextH ws_serializer_init()
 		u_free(serializercntx);
 		return NULL;
 	}
-	u_init_lock(serializercntx);
+	u_init_lock(&serializercntx->lock);
```

Comment:
Ugh, and `ws_serializer_cleanup` is also broken in a similar way (will fix later)